enterprise/server/remote_execution/containers/firecracker/firecracker_test.go

```diff
@@ -798,10 +802,14 @@ func TestFirecracker_RemoteSnapshotSharing(t *testing.T) {
 	require.NotEmpty(t, res.VMMetadata.GetSnapshotId())
 
 	// Should still be able to start from the original snapshot if we use
-	// a snapshot key containing the original VM's snapshot ID
+	// a snapshot key containing the original VM's snapshot ID.
+	// Note that when using a snapshot ID as the key, we only include the
+	// snapshot_id and instance_name fields.
 	workDirForkOriginalSnapshot := testfs.MakeDirAll(t, rootDir, "work-fork-og-snapshot")
-	originalSnapshotKey := baseVM.SnapshotKeySet().GetBranchKey()
-	originalSnapshotKey.SnapshotId = baseSnapshotId
+	originalSnapshotKey := &fcpb.SnapshotKey{
```

Comment:
There were 2 issues here:
- We weren't doing anything with `originalSnapshotKey` - from the comment we should be setting it in ContainerOpts so that we resume from this specific snapshot
- I forgot to update this logic here when we simplified snapshot-specific keys so that they only include the instance_name + snapshot_id.